gulf.js

```diff
@@ -133,15 +163,15 @@
 	 */
 	const patchLabelBytesIfNeeded = (labelBytes) => {
 		// Define the bytes we want to search for
-		const SEARCH_PATTERN_BYTES = [...'Gulf of America'].map(char => char.charCodeAt(0))
+		const SEARCH_PATTERN_BYTES = [...replacementFull].map(char => char.charCodeAt(0))
 
 		// Constants for special cases
 		const CHAR_CODE_SPACE = " ".charCodeAt(0)
 		const CHAR_CODE_CAPITAL_A = "A".charCodeAt(0)
 		const CHAR_CODE_PARENTH = '('.charCodeAt(0)
 		const CHAR_CODE_CAPITAL_G = 'G'.charCodeAt(0)
-    // \u200B is a zero-width space character. We add it to make the strings the same length
-		const REPLACEMENT_BYTES = [..."Mexico\u200B"].map(char => char.charCodeAt(0))
+    	// \u200B is a zero-width space character. We add it to make the strings the same length
+		const REPLACEMENT_BYTES = [...mapping.replacementCountry + '\u200B'].map(char => char.charCodeAt(0))
```

Comment:
This assumes that the localized replacementCountry has exactly 6 letters, which is not always the case. A .repeat() thing might work?

Reply:
i also wonder: if replacementCountry.length > originalCountry.length, will this break the parsing, since the expected protobuf response layout will change cuz alignment will be thrown off?